client/package.json

```diff
@@ -24,31 +24,32 @@
     "@babel/preset-env": "^7.24.0",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.23.3",
+    "@eslint/js": "^9.38.0",
     "@testing-library/jest-dom": "^6.4.2",
     "@testing-library/react": "^14.2.1",
     "@testing-library/user-event": "^14.5.2",
     "@types/react": "^18.0.38",
     "@types/react-dom": "^18.0.11",
-    "@typescript-eslint/eslint-plugin": "^6.18.1",
-    "@typescript-eslint/parser": "^6.18.1",
+    "@typescript-eslint/eslint-plugin": "^8.46.2",
+    "@typescript-eslint/parser": "^8.46.2",
     "axios": "^1.6.7",
-    "eslint": "^8.56.0",
-    "eslint-config-prettier": "^9.1.0",
+    "eslint": "^9.38.0",
+    "eslint-config-prettier": "^10.1.8",
     "eslint-plugin-prettier": "^5.1.2",
     "eslint-plugin-react": "^7.33.2",
     "jsdom": "^24.0.0",
     "prettier": "^3.1.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "vite": "^5.0.0",
-    "vitest": "^1.4.0"
+    "vite": "^7.1.11",
```

Comment:
double-check that this version exists and is stable - if v7 is less than 3 months old, let's use v6